### PR TITLE
Http transport 4.12

### PR DESCRIPTION
**Pull request by aneeshkp**
Add http support to cloud event proxy

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -44,27 +45,31 @@ import (
...
+	pluginHandler      plugins.Handler
 )
 
 func main() {
 	// init
 	common.InitLogger()
 	flag.StringVar(&metricsAddr, "metrics-addr", ":9091", "The address the metric endpoint binds to.")
 	flag.StringVar(&storePath, "store-path", ".", "The path to store publisher and subscription info.")
-	flag.StringVar(&amqpHost, "transport-host", "amqp:localhost:5672", "The transport bus hostname or service name.")
+	flag.StringVar(&transportHost, "transport-host", "amqp:localhost:5672", "The transport bus hostname or service name.")
```
Comment (jzding) [inline cmd/main.go:70]: How about set default value to `localhost:9043` for local http testing?
`flag.StringVar(&transportHost, "transport-host", "localhost:9043", "The transport bus hostname or service name.")`

Comment (aneeshkp) [reply cmd/main.go:70]: For http it should never set to localhost, it should be k8s service url

File: cmd/main.go [hunk 2]
```diff
@@ -44,27 +45,31 @@ import (
...
 	flag.IntVar(&apiPort, "api-port", 8089, "The address the rest api endpoint binds to.")
+	flag.StringVar(&httpEventPublisher, "http-event-publishers", "", "Comma separated address of the publishers available.")
```
[the remark this answers is not in the document]
Comment (aneeshkp) [reply cmd/main.go:72]: Same here, for http it should never set to localhost, it should be k8s service url

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**